crates/parser/src/cli.rs

```diff
@@ -0,0 +1,29 @@
+use std::fs;
```

Comment:
```suggestion
use fs_err as fs;
```
Adds file paths to `io::Error`s which is super convenient and avoids a bunch of `unwrpa_or(format!("{path}", path=path.display()))`